components/camel-ftp/src/main/java/org/apache/camel/component/file/remote/FtpConfiguration.java

```diff
@@ -35,6 +35,8 @@
     private String activePortRange;
     @UriParam(label = "producer,advanced")
     private String chmod;
+    @UriParam(label = "consumer,advanced")
+    private boolean isDirectoryParseResultAbsolute;
```

Comment:
dont use isXXX as the name. Naming in IT is hard, and to come up with a good name.
Maybe the name should be something like `handleDirectoryParserAbsoluteResult` ?